### ISSUE TITLE
Common config directory for multiple projects

### ISSUE THREAD

**Issue by dimmel82** [qoomon/maven-git-versioning-extension#23]
Hi,
I have a sort-of "multi-project" configuration with an aggregate pom.xml that defines 5 modules, e.g.
```
...
<groupId>my.group</groupId>
<artifactId>aggregate<artifactId>
<version>1.0-SNAPSHOT</version>
<packaging>pom</packaging>

<modules>
  <module>m1</module>
  <module>m2</module>
  ...
  <module>m5</module>
</modules>
```
However the m1-m5 projects do not have the aggregate pom as parent, because they inherit the plugins behaviour and properties from an organizational-level parent pom (which means that the solution of issue #6 does not apply in this case).
So, to make the maven-git-versioning-extension plugin to work without copying the extensions.xml and maven-git-versioning-extension.xml to all the projects I would need an additional optional parameter for the plugin to override the `${project.basedir}/.mvn` directory with a common path.
I would be glad to give it a try and create a pull request if you think that this is a valid requirement.

**Comment by qoomon** [comment]
looking forward for the pull request.
an other solution would be to create a link `.mvn` in each related project that points it to the `.mvn` directory of the root project

**Comment by dimmel82** [comment]
Well, I opened this issue as an enhancement based on what I read at issue #6.
But after testing the plugin on my projects I was pleasantly surprised to find out that it works like a charm without any modification.
I created the extensions.xml and maven-git-versioning-extension.xml in _\<root dir>/.mvn_ at the root directory and the version was correctly modified both for the root _aggregate_ project and the sub-module projects.

So, I'm closing this issue and btw I would like to thank you for this really helpful plugin.
